lib/api/2.0/skus.js

```diff
@@ -0,0 +1,168 @@
+// Copyright 2016, EMC Inc.
+
+'use strict';
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var _ = injector.get('_'); // jshint ignore:line
+var SkuPack= injector.get('Http.Services.SkuPack');
```

Comment:
In JavaScript name convention, the variable name with its first letter being uppercase, it means it is a `Class`. Since `Http.Services.SkuPack` is an instance, so please rename the varaible to `skuPack`.